feat(locale): add Swedish(sv) counties

johanholmerin opened the pull request:
Based on the list at https://sv.wikipedia.org/wiki/Sveriges_l%C3%A4n#Lista_%C3%B6ver_Sveriges_l%C3%A4n

Note: although län is the administrative area that translates to [counties](https://en.wikipedia.org/wiki/Counties_of_Sweden), it would also be useful to have support for [municipalities](https://en.wikipedia.org/wiki/Municipalities_of_Sweden)(kommuner), since that's whats used for addresses in Sweden.

johanholmerin committed:
feat(locale): add Swedish(sv) counties

matthewmayer commented:
Generally state() returns the "level 1 administrative areas" and county() returns the "level 2 administrative areas" regardless of what they are actually called in that language. 

Does that make sense for sv?

johanholmerin committed:
Switch sv counties to municipalities

Based on official list at
https://skr.se/skr/tjanster/kommunerochregioner/kommunerlista.1246.html

johanholmerin commented:
Thanks @matthewmayer, that helped make things clearer. I've switched to using municipalities(kommuner) instead(based on the official list at https://skr.se/skr/tjanster/kommunerochregioner/kommunerlista.1246.html). The län were already available as state().

ST-DDT committed:
Merge branch 'next' into sv-county
